Update cloud-import-a-project-by-git-url.md (#1058)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url.md

In the newly created Repository details page, you'll see a "Deploy Key" field. Copy the contents of this field as you'll need it for step 3.

> **Note: This dbt Cloud generated public key will be the only key that will work in the next step. Any other key that has been generated outside of dbt Cloud will not work.**

Reviewer: ```suggestion
**Note:** The dbt Cloud-generated public key is the only key that will work in the next step. Any other key that has been generated outside of dbt Cloud will not work.
```
